website/src/Link.js

```diff
@@ -1,5 +1,8 @@
 import React, { PropTypes, Component } from 'react';
 
+const isModifiedEvent = (event) =>
```

Comment:
Actually I think click event is modified by pressing modifier key. My opinion, could be wrong :)